docs/source/getting-started/tutorial.rst

```diff
@@ -67,3 +72,8 @@ the job, use the :code:`sky cancel <cluster_name> <job_id>` command (refer to :r
 
 After training, :ref:`transfer artifacts <sync-code-artifacts>` such
 as logs and checkpoints using familiar tools.
+
+.. tip::
+
+  Feel free to copy-paste the YAML above and start plugging in
+  your own project.
```

Comment:
```suggestion
  Feel free to copy-paste the YAML above and start plugging it into
  your own project.
```
nit

Reply:
I meant to say the project commands should be plugged into this YAML, not the other way. Reworded to "customize it" to avoid confusion.